ci/scripts/run_build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+build_default() {
+    ./ci/scripts/build_projects.sh
+    ./ci/scripts/cppcheck.sh
+}
+
+build_astyle() {
+    ./ci/scripts/astyle.sh
```

Comment:
this is fine as-is; 
but only as a note, you could also just include the scripts with `. ./ci/scripts/astyle.sh` or `source ./ci/scripts/astyle.sh`  [ the latter only works on `bash` ]

the difference is that
* `./ci/scripts/astyle.sh` will run the script as a separate executable
* `. ./ci/scripts/astyle.sh` will include the script code into this script and run it as part of the same script